themes/ddbasic/scripts/menu.js

```diff
@@ -153,7 +153,7 @@
         return;
       }
 
-      if (main_menu_wrapper[0].offsetTop != secondary_menu_wrapper[0].offsetTop) {
+      if (main_menu_wrapper[0].offsetTop !== secondary_menu_wrapper[0].offsetTop) {
```

Comment:
JS strict comparison is requested by coding standards.

Reply:
Let's leave this for ding2 team to fix.